packages/tasks/src/model-libraries.ts

```diff
@@ -1261,6 +1261,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"model_vae_fp16.pt"`,
 		snippets: snippets.threedtopia_xl,
 	},
+	"HunyuanImage-2.1":{
+		prettyLabel: "HunyuanImage-2.1",
+		repoName: "HunyuanImage-2.1",
+		repoUrl: "https://huggingface.co/tencent/HunyuanImage-2.1",
+		countDownloads: `path_filename:"hunyuanimage2.1.safetensors"`,
```

Comment:
looking at the models listed on https://huggingface.co/models?other=HunyuanImage-2.1, I can't find a `"hunyuanimage2.1.safetensors"` file in them. Best would be to adopt a common structure in these repos and then chose a file to count the downloads from.